packages/contracts/test/dollar/core/DollarMintExcess.t.sol

```diff
@@ -77,8 +75,9 @@ contract DollarMintExcessTest is LocalTestHelper {
         uint256 _expectedExchangeAmt
     ) public {
         vm.prank(admin);
-        UbiquityDollarManager(dollarManagerAddress)
-            .setStableSwapMetaPoolAddress(_metaPoolAddress);
+        UbiquityDollarManager(dollarManagerAddress).setStableSwapMetaPoolAddress(
+            _metaPoolAddress
+        );
         vm.mockCall(
             _metaPoolAddress,
             abi.encodeWithSelector(IMetaPool.exchange.selector),
```

Comment:
I just realized that GitHub UI italicizes a handful of words. I count `super` and `selector` already.